### DISCUSSION TITLE
HEDA app Installation

### DISCUSSION THREAD

**Discussion by Bluerrror** [tethysplatform/tethys#644] [Q&A]
Hi there

I installed the tethys platform and when I want to install HEDA app, the install.yml is created and I filled the dependencies but again I have a problem to install the app as the following error of PFA is arise. Can anyone help me?

<img width="566" alt="Error" src="https://user-images.githubusercontent.com/15969397/107005549-17b5fe80-67a5-11eb-88cf-c642a7011bb7.png">

**Comment by swainn** [comment]
Hi @shahab271069, I'm guessing the HEDA app was developed using Tethys 2? There are a few additional steps you need to perform to migrate a  Tethys 2 app to be compatible with Tethys 3. Take a look at our migration guide here:

http://docs.tethysplatform.org/en/stable/whats_new/app_migration.html

**Reply by swainn** [reply]
Hi @shahab271069 did this help solve your problem?

**Reply by Bluerrror** [reply]
Sorry for delay. Actually I will check and inform you soon. 

Thank you for your help.

**Comment by Bluerrror** [comment]
Hi Nathan

Thank you for your help. But it is not working.

Best
Shahab

On Tue, Mar 2, 2021 at 5:41 AM Nathan Swain <notifications@github.com>
wrote:

> Hi @shahab271069 <https://github.com/shahab271069> did this help solve
> your problem?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tethysplatform/tethys/discussions/644#discussioncomment-420274>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADZ2Y5LXRTLWFZJVBYQ7XG3TBRCOPANCNFSM4XEJUEOQ>
> .
>


-- 
Ph.D Candidate: School of Engineering, Department of Civil and
Environmental Engineering, Shiraz University, Shiraz, Iran
Address: School of Engineering, Department of Civil and Environmental
Engineering, Shiraz University, Shiraz, Iran
Telephone: +989183497499
Alternative E-mail: s.shojaeezadeh@shirazu.ac.ir
Google Scholar Profile:
https://scholar.google.com/citations?user=5wfrVUwAAAAJ&hl=en
LinkedIn Profile: https://www.linkedin.com/in/shahab-shojaee-b6ba5a90/
Researchgate Profile:
https://www.researchgate.net/profile/Shahab_Aldin_Shojaeezadeh
